src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BALL_NUMBER = 100;
```

Comment:
Please be careful with bound of `random.nextInt(MAX_BALL_NUMBER)`. This method call returns a pseudorandom, uniformly distributed int value between 0 (inclusive) and n (**exclusive**).